data/CosmosDbSeeder/items/local/booking_data/autotest_booking_01.json

```diff
@@ -0,0 +1,27 @@
+{
+  "reference": "autotest-booking-reference-01",
+  "from": "2025-08-05T09:30:00",
+  "duration": 10,
+  "service": "RSV:Adult",
+  "created": "2025-09-03T12:58:28.701582+00:00",
```

Comment:
It probably doesn't matter, but just to be extra safe can we make the Created date earlier than the From date, as a user wouldn't have been able to book an appointment in the past :)